src/screens/home/components/HomeView.js

```diff
@@ -109,6 +112,15 @@ const HomeView = ({ route, navigation, setLauncherContext, setBarStyle }) => {
     }
   }, [uri, client, route, navigation, session])
 
+  useEffect(() => {
+    const lockRedirect = async () => {
+      const shouldRedirect = await getData(StorageKeys.AutoLockEnabled)
+      shouldRedirect && navigate(routes.lock)
+    }
+
+    void lockRedirect()
+  }, [])
```

Comment:
With this effect, we will have to be sure that the home cannot be re-mount during app's lifecycle. Hope we won't encounter this kind of side effect in the future :sweat_smile:

Reply:
This would be a terrible bug indeed, but I can try to find an out-of React storage (something in RAM, not asyncStorage) to use as a tag to failproof this so it doesn't ever fire twice

Reply:
So you didn't find any way to call the home/lock simultaneous navigation from the bootstrap?

Reply:
That's what it's doing. As soon as Home get to this useEffect, it redirects to lockScreen. Some time later, the intent will hideSplashScreen while LockScreen is already rendering/rendered

Reply:
What I meant was "from the useAppBootstrap.js file". This would ensure that home's rerender cannot have side effect on this.

Reply:
For now the only way to have simultaneous renders that I found working is doing it this way; navigating into the nested route then navigating to the modal ASAP => this is a trick but it will actually render the nested route and the dialog route at the same time